jalantechnologies/boilerplate-mern
# Add Signup page to create a new account using Tailwind template

## Description
This PR introduces the signup page that have been created using tailwind to create a new account.

## Database schema changes
- NA

## Tests
### Automated test cases added
- NA

### Manual test cases run
> created a new account and logged in > working properly

> try logging with non existing account or creating a new account with an existing mail throws error as expected

https://github.com/jalantechnologies/boilerplate-mern/assets/74129325/b802a01d-0227-4522-a714-b1c80e4fc6f9
